website/moneybirdsynchronization/services.py

```diff
@@ -0,0 +1,297 @@
+from events.models.event import Event
+from payments.models import Payment
+from members.models import Member
+from moneybirdsynchronization.models import Contact
+from moneybirdsynchronization.administration import HttpsAdministration, Administration
+from moneybirdsynchronization import emails
+from thaliawebsite import settings
+from django.template.defaultfilters import date
+
+import datetime
+
+class MoneybirdAPIService():
+
+    def __init__(self):
+        api = HttpsAdministration(settings.MONEYBIRD_API_KEY, settings.MONEYBIRD_ADMINISTRATION_ID)
```

Comment:
I would make `api` private, so refactor to `_api` or even `__api`, as you don't want allow others to make calls to it